src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
```

Comment:
There are enum for color representation, I think it is preferable to use it instead of string